Documentation/CRDs/ceph-client-crd.md

```diff
@@ -41,3 +41,146 @@ spec:
 ### Prerequisites
 
 This guide assumes you have created a Rook cluster as explained in the main [Quickstart guide](../Getting-Started/quickstart.md).
```

Comment:
Would you mind moving the prerequisites section above the "Creating Ceph User" section while you are touching this file? It should be first.

Reply:
- [ ] move prereq section to before creating ceph user